electron-app/src/views/settings/General.vue

```diff
@@ -125,6 +124,51 @@
             </v-switch>
           </v-card-text>
         </v-card>
+        <v-card class="mt-5">
+          <v-card-title>User Account & Security</v-card-title>
+          <v-form ref="form">
+            <v-card-text>
+              <revealable-input
+                v-model="currentPassword"
+                class="settings-general__account-and-security__fields__current-password"
+                label="Current Password"
+                :rules="passwordRules"
+              ></revealable-input>
+              <revealable-input
+                v-model="newPassword"
+                class="settings-general__account-and-security__fields__new-password"
+                label="New Password"
+                :rules="passwordRules"
+              ></revealable-input>
+              <revealable-input
+                v-model="newPasswordConfirm"
+                class="settings-general__account-and-security__fields__new-password-confirm"
+                label="Confirm New Password"
+                icon="fa-repeat"
+                :rules="passwordConfirmRules"
+                :error-messages="errorMessages"
+              ></revealable-input>
+            </v-card-text>
+            <v-card-actions>
+              <v-btn
+                depressed
+                class="settings-general__account-and-security__buttons__change-password"
+                color="primary"
+                type="submit"
```

Comment:
this causes the page to redirect when you click the button. I am not even sure the changePassword manages to run before the redirect.

Reply:
Good catch. It wasn't happening elsewhere because the button wasn't in a `<v-form>` component, but obviously here a submit is expected to redirect.